Week-1/design-thinking-assignment/Design Thinking Assignment.md

```diff
@@ -0,0 +1,20 @@
+### Design Thinking Problem – Two perpendicular roads meet. Both roads have one lane of traffic traveling in opposite directions. ###
+
+Here I would like to list the essential details to keep in mind while constructing this road intersection.
+
```

Comment:
I would suggest thinking in context of cost and return of value. There are various kinds of intersection out in the world. Each one solves something and adds something. In most situations given that time is available, it is better to prepare three cases , 1. Where everything goes right - God of luck and money thinks that you are their successor. 2. A situation close to ground reality where not everyone will agree, budget will be tight and you might be required to express that "I am only a human being". 3. Rajpal yadav in this clip https://www.youtube.com/watch?v=PU1kvaWwpdY puts it succinctly,